src/GeositeFramework/region.json

```diff
@@ -72,9 +72,9 @@
         "layer_selector",
         "custom_layer_selector",
         "zoom_to",
-        "full_extent",
         "measure",
         "legend_display",
+        "full_extent",
```

Comment:
We'll have to remind TNC that this is all they need to do to accomplish this in their own regions.